apispec/openapi.json

```diff
@@ -5201,32 +5201,32 @@
       "transport.id": {
         "title": "transportId",
         "type": "integer",
-        "description": "Indicates an transport id, it is a unique integer number in HDT System."
+        "description": "Indicates a transport id. It is a unique integer number in the HDT system."
       },
       "transport.code": {
         "title": "transportCode",
         "type": "string",
-        "description": "if transport type of this transport is ipport, it uses this form ‘targetDomain:targetPort’ to represent. if transport type of this transport is vpn or socks5, it uses ‘userAccount’ to represent."
+        "description": "If transport type of this transport is ipport, it uses this form ‘targetDomain:targetPort’ to represent. If transport type of this transport is vpn or socks5, it uses ‘userAccount’ to represent."
       },
       "transport.type": {
         "title": "transportType",
         "type": "string",
-        "description": "Indicates transport type which this transport is belonged to."
+        "description": "Indicates the transport type to which this transport belonged."
       },
       "transport.name": {
         "title": "transportName",
         "type": "string",
-        "description": "Represent a transport."
+        "description": "Represents a transport."
       },
       "transport.targetDomain": {
         "title": "targetDomain",
         "type": "string",
-        "description": "A domain/IP refers to the destination server of customer application, HDT would access this domain/IP to get data from original server. When transport type is ipport, this parameter is valid and required. It can be a domain name or an ip. The length of it must be less than or equal to 250."
+        "description": "A domain/IP refers to the destination server of the customer application. HDT would access this domain/IP to get data from the original server. When transport type is ipport, this parameter is valid and required. It can be a domain name or an IP address. The length of it must be less than or equal to 250."
```

Comment:
Change the beginning to "A domain or IP referring to the destination server of your application."
Again, I don't really like referring to "the customer" when the target audience of our API documentation is our customer.
Also, I suggest the last sentence be "Its length must be no more than 250 characters."